packages/sdk/src/build/build-get-block-header.js

```diff
@@ -1,5 +1,10 @@
 import {pipe, Ok, makeGetBlockHeader} from "../interaction/interaction.js"
 
+/**
+ * @description - A builder function that returns the interaction to get a block header
+ * @param {boolean} isSealed - Whether or not the block should be sealed
```

Comment:
Technically this is a type mismatch. Needs to be `boolean | null`.

Reply:
Good catch, made them optionals